src/lib/styles/global/button.scss

```diff
@@ -132,4 +134,44 @@ button {
       filter: contrast(1.25);
     }
   }
+
+  &.card {
```

Comment:
Stupid question, but where's the background?

Shouldn't there be a place that sets background to `transparent` when not focus, nor active, no hover?

Reply:
Default = no background = transparent